pkg/apis/networkaddonsoperator/v1alpha1/zz_generated.deepcopy.go

```diff
@@ -5,6 +5,7 @@
 package v1alpha1
 
 import (
+	v1 "github.com/openshift/custom-resource-status/conditions/v1"
```

Comment:
conditionsv1

Reply:
As Sebastian commented, it is generated and we have no control over the name.

Reply:
ack